sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/command_component.py

```diff
@@ -48,6 +48,7 @@ class Meta:
             ]
         ),
     )
+    properties = fields.Dict(keys=fields.Str(), values=fields.Raw())
```

Comment:
what is fields.Raw stands for?

Reply:
It will pass whatever user wrote in YAML with original format. e.g. true -> bool, "str" -> str, 1 -> int